vars/setupCommonEnvironmentVariables.groovy

```diff
@@ -48,21 +48,26 @@ def call() {
             env.UPGRADE_PAUSE_TIME = ""
             env.UPGRADE_DELAY_TIME = ""
 
-
-            //E2e Variables
-            env.E2E_GIT = "https://github.com/openshift/origin"
-            env.E2E_BRANCH="release-${env.OCP_RELEASE}"
-            if (OCP_RELEASE == "4.5" || OCP_RELEASE == "4.6" ) {
-                env.E2E_EXCLUDE_LIST = "https://raw.github.ibm.com/redstack-power/e2e-exclude-list/${env.OCP_RELEASE}-powervm/ocp${env.OCP_RELEASE}_power_exclude_list.txt"
+            if ( env.ODF_VERSION ) {
+                env.INSTANCE_NAME = "jenkins-odf"
```

Comment:
Stick to rdr-cicd at the starting,this to identify the CI resources..  something like `rdr-cicdodf` ?